olexandr.vakulinskiy/HW2/Task1/tamagotchi.rb

```diff
@@ -0,0 +1,115 @@
+class Tamagotchi
+  attr_accessor :name, :health, :hunger, :thirst, :fatigue
+
+  def initialize(name)
+    @name = name
+    @health = 10
+    @hunger = 2
+    @thirst = 2
+    @fatigue = 0
+  end
+
+  def check_health
+    if @hunger.zero? || @thirst.zero?
+      @health += 1
+    elsif @hunger >= 7 || @thirst >= 7
+      @health -= 1
+    end
+
+    case @fatigue
+    when 10
+      @health = 0
+    when 7...10
+      @health -= 2
+    end
```

Comment:
I meant the other, you need to merge both conditions in one case 🙂 (if I understand correctly condition)

Reply:
I'm not sure how to merge it, because in case must be different characteristic

Reply:
@AlexanderVak You're right, I didn't notice that different attributes are checked 👍